lib/tsconfig.test.json

```diff
@@ -0,0 +1,4 @@
+{
+    "extends": "./tsconfig.json",
+    "include": ["test/"]
```

Comment:
@allejo Heads up, if you are planning to continue working on this PR you might want to update the branch with the develop since recently we have changed the file structure significantly and this file for example won't be needed.

Reply:
Appreciate the heads up! Just noticed the changes as I brought in `develop` to fix the web-test-runner issue.